src/GenericSpectator/ProjectSettings/ProjectSettings.asset

```diff
@@ -0,0 +1,881 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!129 &1
+PlayerSettings:
+  m_ObjectHideFlags: 0
+  serializedVersion: 15
+  productGUID: 674eb9d4c0e4c4e4691c31ed7ecfcebd
+  AndroidProfiler: 0
+  AndroidFilterTouchesWhenObscured: 0
+  AndroidEnableSustainedPerformanceMode: 0
+  defaultScreenOrientation: 0
+  targetDevice: 2
+  useOnDemandResources: 0
+  accelerometerFrequency: 60
+  companyName: Microsoft
+  productName: SpectatorView.GenericSpectator
```

Comment:
>SpectatorView.GenericSpectator [](start = 15, length = 30)

@matthejo, OK with this product name?

Reply:
We should add a .Unity extension to this project's folder naming. long term src may contain other middleware code bases

Reply:
Thanks!

---
In reply to: [337246501](https://github.com/microsoft/MixedReality-SpectatorView/pull/250#discussion_r337246501) [](ancestors = 337246501)